spec/index.bs

```diff
@@ -8403,50 +8404,58 @@ a list of vertices to process for each instance.
 <div algorithm>
     <dfn abstract-op>resolve indices</dfn>(drawCall, state)
 
-        **Arguments:**
-            - |drawCall|: The draw call parameters.
-            - |state|: The active [=RenderState=].
-
-        **Returns:** list of integer indices.
-
-        1. Let |vertexIndexList| be an empty list of indices.
-        1. If |drawCall| is an indexed draw call:
-            1. initialize the |vertexIndexList| with |drawCall|.indexCount integers.
-            1. for |i| in range 0 .. |drawCall|.indexCount (non-inclusive):
-                1. let |vertexIndex| be [$fetch index$](|i| + |drawCall|.firstIndex,
-                    |state|.[=RenderState/indexBuffer=].buffer, |state|.[=RenderState/indexBuffer=].offset,
-                    |state|.[=RenderState/indexBuffer=].format) + |drawCall|.baseVertex
-                1. append |vertexIndex| to the |vertexIndexList|
-        1. Otherwise:
-            1. initialize the |vertexIndexList| with |drawCall|.vertexCount integers.
-            1. assign the |vertexIndexList| item |i| to be |drawCall|.firstVertex + |i|
-        1. Return |vertexIndexList|.
-
-        Note: in case of indirect draw calls, the `indexCount`, `vertexCount`,
-        and other properties of |drawCall| are read from the indirect buffer
-        instead of the draw command itself.
-
-        Issue: specify indirect commands better.
+    **Arguments:**
```

Comment:
Very unfortunate that indentation changes are mixed with the logic change here. I want to see what changed, and now github can't show me this

Reply:
Hiding whitespace changes on GitHub works fine for me:
https://github.com/gpuweb/gpuweb/pull/1835/files?diff=split&w=1